.gitignore

```diff
@@ -1,20 +1,29 @@
-# Ignore generated folders
+# dependencies
 node_modules
-es
-lib
-storybook-static
+.npmrc
 
-# Logs
+# log files
 *.log
 
-# Editor
+# docs
+.gh-pages
 .DS_Store
-*.swp
-.idea
-.vscode
 
-# Coverage directory used by tools like Istanbul
-coverage
+# distribution files
+lib
+es
+storybook-static
+/scss
+
+# a11y files
+a11y/results
 
-# Generated by npm@5, but project currently uses Yarn
+# we yarn folks here
 package-lock.json
```

Comment:
Are you all using Yarn or npm? If npm, should the lockfile be ignored?

Reply:
@joshblack Using npm - will remove this from gitignore